pom.xml

```diff
@@ -61,6 +61,7 @@
         <hibernate.validator.version>6.0.16.Final</hibernate.validator.version>
         <immutables.version>2.7.1</immutables.version>
         <jackson.databind.nullable.version>0.1.0</jackson.databind.nullable.version>
+        <jackson.databind.version>2.9.9.1</jackson.databind.version>
         <jackson.version>2.9.9</jackson.version>
```

Comment:
They didn't release 2.9.9.1 versions for the other jackson dependencies, so I had to create a new version here.